src/gui/DemuxerThr.cpp

```diff
@@ -907,7 +907,7 @@ void DemuxerThr::emitInfo()
                     videoStreams += "<li><b>" + tr("title") + ":</b> " + streamInfo->title + "</li>";
                 if (!streamInfo->codec_name.isEmpty())
                     videoStreams += "<li><b>" + tr("codec") + ":</b> " + streamInfo->codec_name + "</li>";
-                videoStreams += "<li><b>" + tr("size") + ":</b> " + QString::number(streamInfo->width) + "x" + QString::number(streamInfo->height) + "</li>";
+                videoStreams += "<li><b>" + tr("resolution") + ":</b> " + QString::number(streamInfo->width) + "x" + QString::number(streamInfo->height) + "</li>";
```

Comment:
It is a video frame size, so it is a good text.

Reply:
“Size” has more than one meanings. I think “resolution” is more straightforward than it. If you insist, I can revert this string.